### PR TITLE
RUNTIME-938 Added security implications for unlimiting the working

### PR DESCRIPTION
**Pull request by arnitkun**
Added the information regarding the side effects of allowing postman to access files outside its working directory.

### CODE REVIEW

File: v6/postman/launching_postman/settings.md [hunk 1]
```diff
@@ -22,7 +22,14 @@ Postman tries to minimize the number of settings you have to change, so we estab
...
+   * **Working Directory:** If you want Postman to persist your file paths, then you must save your files in Postman's default working directory. When you work with files in form-data request body and binary file body, save them in this directory to let Postman persist your file's path relative to the working directory. This means your files loaded from within the working directory run smoothly across devices if other users use the same files on their devices. Postman flags a warning for files that are not stored in this directory. 
+
+   However, unlimiting the working directory can have some unintended security issues as follows:
+   1. It is aginst the general principle of security to give system wide access to a program as it exposes a user's system to all types of threats.
+   2. To further elaborate the above statement, restricting the working directory would prevent saftey issues arising when an external file(s) is used. For example, a collection that the user has obtained from the internet. As the user may or may not have proper information about the collection and as such may not understand if the collection serves some other hidden function.
```
Comment (kaustavdm) [inline v6/postman/launching_postman/settings.md]: - Spelling error: "safety".
- Grammar: "when external files are used"

Overall, rewrite this paragraph to make it sound good. It reads broken.

File: v6/postman/launching_postman/settings.md [hunk 2]
```diff
@@ -22,7 +22,14 @@ Postman tries to minimize the number of settings you have to change, so we estab
...
+   3. While Postman can take absolute file paths for it's use, such absolute paths when shared may break the workflow for the user it is shared to.
```
Comment (kaustavdm) [inline v6/postman/launching_postman/settings.md]: - Rephrase.
- Grammar: "its"

File: v6/postman/launching_postman/settings.md (hunk 1 above)
Comment (arnitkun) [reply v6/postman/launching_postman/settings.md]: corrected

File: v6/postman/launching_postman/settings.md (hunk 2 above)
Comment (arnitkun) [reply v6/postman/launching_postman/settings.md]: done

**Pull request merged**